docs/pages/installation.mdx

```diff
@@ -107,21 +107,193 @@ information on obtaining Teleport binaries compatible with Teleport Cloud.
+container. 
+
+For example, if you are running the Teleport Proxy Service on a container, you
+need to mount the directory containing TLS credentials to your Teleport
+container, then assign the following fields in the container's configuration
+file to the appropriate paths:
+
+```yaml
+proxy_service:
+  https_keypairs:
+  - key_file: /my/path/key.pem
+    cert_file: /my/path/cert.pem
+```
+
+See the Teleport [Configuration Reference](reference/config.mdx) for whether a
+field you would like to assign requires a file path.
+
+#### Ports
+
+A single Teleport process can run multiple services, each of which listens on a
+specific set of ports depending on your configuration. See our [Networking
+Reference](reference/networking.mdx#ports) for the ports on your Teleport
+container to expose.
+
+### Example of running a Teleport container
+
+In this example, we will show you how to run the Teleport Auth and Proxy
+Services on a local Docker container using Teleport Community Edition. 
+
+Since this container uses a self-signed certificate, we do not recommend using
+this configuration to protect any infrastructure outside your workstation. You
+can, however, join other local Docker containers to it using the [token
+method](./management/join-services-to-your-cluster/join-token.mdx).
+
+First, create directories in your home directory to mount to the container. The
+Teleport container will write its configuration and data to these directories:
+
+```code
+$ mkdir -p ~/teleport/config ~/teleport/data
+```
+
+Run `teleport configure` from the Teleport container to generate a configuration
+file. This sets the container's name to `localhost` so your browser can trust
+the Proxy Service's self-signed TLS certificate:
+
+```code
+$ docker run --hostname localhost --rm \
+  --entrypoint=/usr/local/bin/teleport \
+  <Var name="(=teleport.latest_oss_docker_image=)" /> configure --roles=proxy,auth > ~/teleport/config/teleport.yaml
+```
+
+Start Teleport on your container:
+
+```code
+$ docker run --hostname localhost --name teleport \
+  -v ~/teleport/config:/etc/teleport \
+  -v ~/teleport/data:/var/lib/teleport \
+  -p 3025:3025 -p 3080:3080 \
+  <Var name="(=teleport.latest_oss_docker_image=)" />
+```
+
+From there, open another terminal and make sure your Teleport container's web
+API is functioning as intended:
+
+```code
+$ curl --insecure https://localhost:3080/webapi/ping
+{"auth":{"type":"local","second_factor":"otp","preferred_local_mfa":"otp","local":{"name":""},"private_key_policy":"none","device_trust_disabled":true,"has_motd":false},"proxy":{"kube":{"enabled":true,"listen_addr":"0.0.0.0:3080"},"ssh":{"listen_addr":"0.0.0.0:3080","tunnel_listen_addr":"0.0.0.0:3080","web_listen_addr":"0.0.0.0:3080"},"db":{"postgres_listen_addr":"0.0.0.0:3080","mysql_listen_addr":"0.0.0.0:3080"},"tls_routing_enabled":true},"server_version":"12.1.5","min_client_version":"11.0.0","cluster_name":"localhost","automatic_upgrades":false}
```

Comment:
Consider piping this through `jq` for more human-friendly output. You don't necessarily need to include it in the example command, just adjust the output so it's more docs-friendly.